CriticalMass/UIView+Autolayout.swift

```diff
@@ -22,4 +22,15 @@ extension UIView {
             widthAnchor.constraint(equalToConstant: size.width)
         ])
     }
+    
+    func addLayoutsSameSizeAndOrigin(in view: UIView) {
+        translatesAutoresizingMaskIntoConstraints = false
+        
+        view.addConstraints([
+            NSLayoutConstraint(item: self, attribute: .width, relatedBy: .equal, toItem: view, attribute: .width, multiplier: 1, constant: 0),
```

Comment:
I would opt in again for anchors here. Even Apple does not like its convenience method 😄

> Unlike the approach taken by the layout anchor API, you must specify a value for each parameter, even if it doesn’t affect the layout. The end result is a considerable amount of boilerplate code, which is usually harder to read. For example, the code in Listing 13-2 code is functionally identical to the code in Listing 13-1.

https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/AutolayoutPG/ProgrammaticallyCreatingConstraints.html